mail/migrations/0002_default_email_templates.py

```diff
@@ -0,0 +1,101 @@
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    """
+    It adds default email templates
+    """
+    AutomaticReminderEmail = apps.get_model("mail", "AutomaticReminderEmail")
+    db_alias = schema_editor.connection.alias
+
+    # first email for reminders to pay one week before deadline
+    AutomaticReminderEmail.objects.using(db_alias).create(
+        days_before=7,
+        email_subject="Payment due for {bootcamp_name} Bootcamp",
+        email_body="""Dear {full_name},
+
+Your tuition payment for {bootcamp_name} is due. Your current balance is ${remaining_balance}. \
+Please pay your balance before {payment_due_date}.
+
+You can pay online with a credit card or PayPal at {site_URL}.
+
+If you have submitted the payment by wire transfer, could you please send us the transaction receipt? \
+We will need this it to confirm receipt of funds through the wire department.
+
+Please let us know at your earliest convenience.
+
+
+Thanks & best regards,
+
+Admissions Committee
+MIT Global Entrepreneurship Bootcamps
+        """
+    )
+
+    # second email for reminders to pay 2 days before deadline
+    AutomaticReminderEmail.objects.using(db_alias).create(
+        days_before=2,
+        email_subject="{bootcamp_name} Bootcamp Payment due in 2 days",
```

Comment:
Here we say `{bootcamp_name} Bootcamp` but in the body we just use `{bootcamp_name}`. That might be fine, just want to double check this